release-please-config.json

```diff
@@ -1,7 +1,6 @@
 {
-  "bootstrap-sha": "bc6f822e7a6927b65b63c91021ede10dad73006f",
+  "bootstrap-sha": "47035e3bd4e3ac399319b3421ce86a82f1c1ab21",
```

Comment:
**question(non-blocking):** Shall we remove this line after the next release [as mentioned in the docs](https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#starting-commit)?

Reply:
No, I think it's still used if it can't find a release for a new component, e.g., it will use that as the search limit when we add `plugins/typescript` for the first time. I assume if it's removed entirely then it will instead search the last 500 commits which seems to be its default limit. That's bad for us because the commit which sets every component to `v2.0.0` is included in that batch of the last 500 commits (for now)!

Reply:
So if we removed it after the next release I _think_ we'd run into the same issue the next time we create a new plugin.

Reply:
Makes sense! Now we have this explainer bound to the commit's PR so it isn't removed by someone like me in the future 😄